eng/pipelines/coreclr/perf.yml

```diff
@@ -112,6 +113,156 @@ jobs:
         logicalmachine: 'perftiger'
         javascriptengine: 'javascriptcore'
 
+  # build mono Android scenarios
```

Comment:
Are these changes going into `main` as well?

Reply:
These are not going to be added into main scheduled section for the moment as the non-scheduled versions of these cover the frequency attained here.